java/swagger-petstore/README.md

```diff
@@ -0,0 +1 @@
+The new Petstore sample.
```

Comment:
See readme of other 2.0 inflector based samples and update readme

Reply:
Done!